scripts/docker-dos/dosUrlLocalizer.sc

```diff
@@ -54,7 +54,8 @@ case class MarthaResponse(dos: DosObject, googleServiceAccount: JsObject)
 @main
 def dosUrlResolver(dosUrl: String, downloadLoc: String) : Unit = {
   val dosResloverObj = for {
-    marthaResObj <- resolveDosThroughMartha(dosUrl)
+    marthaUrl <- Uri.fromString(sys.env("MARTHA_URL")).toTry
+    marthaResObj <- resolveDosThroughMartha(dosUrl, marthaUrl)
```

Comment:
what happens if this environment variable isn't defined?

Reply:
It throws `java.util.NoSuchElementException: key not found: MARTHA_URL`